src/sentry/integrations/github/webhook.py

```diff
@@ -38,43 +38,62 @@ def get_external_id(username):
 
 
 class Webhook(object):
-    def __call__(self, event, organization=None):
+    def _handle(self, event, organization, repo):
         raise NotImplementedError
 
+    def __call__(self, event):
+        integration = Integration.objects.get(
+            external_id=event['installation']['id'],
+            provider='github',
+        )
+        # TODO(maxbittker) select org directly to make this pattern scale to many
+        # orgs on one integration
```

Comment:
do you mean select repo directly? because yeah, it seems like you could optimize this by selecting all the repos from the different orgs at once. it's ok to do that in a follow up pr though

Reply:
gonna tackle this in the next pr where I add the tests for this behavior